### PR TITLE
VIITE-1119_conversion-table has to be defined in roadaddressimport

### PR DESCRIPTION
**Pull request by oappicgi**
added requirment for conversiontable name as input to importroadaddress batch.
will be read as second argument

### CODE REVIEW

**Comment by oappicgi** [comment]
hmm doesnt seem to work in task runner (even though it works from command line)

File: digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala [hunk 1]
```diff
@@ -77,29 +74,32 @@ object DataFixture {
...
-      val importOptions = ImportOptions(
-        onlyComplementaryLinks = false,
-        useFrozenLinkService = dr2properties.getProperty("digiroad2.VVHRoadlink.frozen", "false").toBoolean,
-        geometryAdjustedTimeStamp.toLong, conversionTable,
-        onlyCurrentRoads = dr2properties.getProperty("digiroad2.importOnlyCurrent", "false").toBoolean)
-      dataImporter.importRoadAddressData(Conversion.database(), vvhClient, importOptions)
+      importTableName match {
+        case None => // shouldn't get here because args size test
+          println(s"****** Import failed! conversiontable name required as second input ******")
+          System.exit(-1)
```
Comment (hestrompa) [inline digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala]: Can we avoid this System.exit(-1)? If we just return nothing I think the result will be the same

Comment (oappicgi) [reply digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala]: k, thought it wouldn't show as error in jenkins if it didn't throw -1. sure i can remove it

Comment (hestrompa) [reply digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala]: I think that it is not wrong at all, I just think we can handle it in a different way :) 
Like using the break() function or returning an exception.

Comment (oappicgi) [reply digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/util/DataFixture.scala]: ok, thanks i will add that =)

**Pull request merged**